spk/sickrage/src/installer.sh

```diff
@@ -65,7 +65,7 @@ postinst ()
     ${VIRTUALENV} --system-site-packages ${INSTALL_DIR}/env > /dev/null
 
     # Clone the repository and configure autoProcessTV
-    ${GIT} clone -q -b ${wizard_fork_branch:=master} ${wizard_fork_url:=git://github.com/SiCKRAGETV/SickRage.git} ${INSTALL_DIR}/var/SickRage
+    ${GIT} clone -q -b ${wizard_fork_branch:=master} ${wizard_fork_url:=https://github.com/SickRage/SickRage.git} ${INSTALL_DIR}/var/SickRage
```

Comment:
Fwiw, that doesn't really apply to Synology devices. We have control, because we provide the Python package as well.